packages/base/boolean.gts

```diff
@@ -35,6 +36,17 @@ export default class BooleanField extends FieldDef {
   static [serialize](val: any) {
     return Boolean(val);
   }
+
+  static async [deserialize]<T extends BaseDefConstructor>(
+    this: T,
+    val: any,
+  ): Promise<BaseInstanceType<T>> {
+    if (val === undefined || val === null) {
+      return false as BaseInstanceType<T>;
+    }
+    return Boolean(val) as BaseInstanceType<T>;
+  }
```

Comment:
Since we may possibly have null in the JSON but may want to display a default value for certain components. We stub a default value via the deserialize hook. This doesn't trigger editing of the card in the constructor.

I think there is compelling use-case for this. For example, like when displaying calendar date